extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -617,7 +624,13 @@ export class SystemLayer {
    */
   onPageIndexUpdate_(index) {
     this.vsync_.mutate(() => {
+      const lastIndex =
+        this.storeService_.get(StateProperty.PAGE_IDS).length - 1;
       this.getShadowRoot().classList.toggle('first-page-active', index === 0);
+      this.getShadowRoot().classList.toggle(
```

Comment:
Can we please prefix these classes with `i-amphtml-`?

Reply:
Done.